pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -449,6 +449,9 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 		}
 
 		if cr.Spec.Pause {
+			if cr.Status.State == AppStateStopping {
+				log.Info("Pausing cluster", "replset", replset.Name, "oldSize", replset.Size, "newSize", 0)
```

Comment:
Please add namespace here for cluster wide mode.

Reply:
I just rebased this branch, with this PR #1129 this logger will inherit the namespace key, so it will be populated.